[WKCI] Make WPE-Cairo bot build with LibWebRTC

dpino opened the pull request:
#### 084d2466a8fe59f2fb5dad438f09e12cb7721f07
<pre>
[WKCI] Make WPE-Cairo bot build with LibWebRTC
<a href="https://bugs.webkit.org/show_bug.cgi?id=303088">https://bugs.webkit.org/show_bug.cgi?id=303088</a>

Reviewed by Carlos Alberto Lopez Perez.

Changeset 300680@main, added &apos;--cmakeargs=-DUSE_GSTREAMER_WEBRTC=FALSE&apos; to the
&apos;additionalArguments&apos; array of &apos;WPE-Cairo-Build-EWS&apos;. However, this extra
argument was never passed to the bot because since 263387@main is necessary
that &apos;additionalArguments&apos; are also added to &apos;VALID_ADDITIONAL_ARGUMENTS_LIST&apos;
for builder bots in order to be applied.

Since we can control CMake flags at bot configuration level, we&apos;ve opted
for never use &apos;additionalArguments&apos; for build flags.

Also, rename &apos;wpe-cairo&apos; to &apos;wpe-cairo-libwebrtc&apos;.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
* Tools/CISupport/ews-app/ews/common/github.py:
* Tools/CISupport/ews-app/ews/views/statusbubble.py:
* Tools/CISupport/ews-build/config.json:
* Tools/CISupport/ews-build/factories.py:
* Tools/CISupport/ews-build/factories_unittest.py:
* Tools/CISupport/ews-build/loadConfig.py:
* Tools/CISupport/ews-build/steps.py:
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: <a href="https://commits.webkit.org/303565@main">https://commits.webkit.org/303565@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/109c9135f1c6e357f2342cc76e8611fdbf76741c

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/132829 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/159/builds/5330 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/43914 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/140361 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/84857 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/134699 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/155/builds/5669 "Built successfully") | [  ~~🛠 mac-AS-debug~~](https://ews-build.webkit.org/#/builders/156/builds/5193 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/101563 "Passed tests") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/68862 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/135775 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/162/builds/3983 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/119004 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/82356 "Passed tests") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/154/builds/3870 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [⏳ 🧪 api-mac-debug](https://ews-build.webkit.org/#/builders/macOS-Tahoe-Debug-API-Tests-EWS "Waiting to run tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/83594 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/112861 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/37122 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/143016 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/153/builds/4999 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/37707 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/109937 "Passed tests") | 
| [✅ 🧪 services](https://ews-build.webkit.org/#/builders/28/builds/132257 "Passed tests") | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/160/builds/5081 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/156/builds/5193 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/110115 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/164/builds/3823 "Passed tests") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/115270 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/58515 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/20593 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/158/builds/5053 "Built successfully") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/33623 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/157/builds/4891 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/163/builds/5143 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/152/builds/5011 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

clopez commented on Tools/CISupport/ews-build/config.json:
```diff
@@ -415,10 +415,9 @@
     {
-      "name": "WPE-Cairo-Build-EWS", "shortname": "wpe-cairo", "icon": "buildOnly",
-      "factory": "WPECairoBuildFactory", "platform": "wpe",
+      "name": "WPE-Cairo-LibWebRTC-Build-EWS", "shortname": "wpe-cairo-libwebrtc", "icon": "buildOnly",
```
you need to update the shortname also at `Tools/CISupport/ews-app/ews/common/github.py` and `Tools/CISupport/ews-app/ews/views/statusbubble.py`

dpino committed:
[WKCI] Make WPE-Cairo bot build with LibWebRTC
https://bugs.webkit.org/show_bug.cgi?id=303088

Reviewed by Carlos Alberto Lopez Perez.

Changeset 300680@main, added '--cmakeargs=-DUSE_GSTREAMER_WEBRTC=FALSE' to the
'additionalArguments' array of 'WPE-Cairo-Build-EWS'. However, this extra
argument was never passed to the bot because since 263387@main is necessary
that 'additionalArguments' are also added to 'VALID_ADDITIONAL_ARGUMENTS_LIST'
for builder bots in order to be applied.

Since we can control CMake flags at bot configuration level, we've opted
for never use 'additionalArguments' for build flags.

Also, rename 'wpe-cairo' to 'wpe-cairo-libwebrtc'.

* Tools/CISupport/build-webkit-org/config.json:
* Tools/CISupport/build-webkit-org/factories_unittest.py:
* Tools/CISupport/ews-app/ews/common/github.py:
* Tools/CISupport/ews-app/ews/views/statusbubble.py:
* Tools/CISupport/ews-build/config.json:
* Tools/CISupport/ews-build/factories.py:
* Tools/CISupport/ews-build/factories_unittest.py:
* Tools/CISupport/ews-build/loadConfig.py:
* Tools/CISupport/ews-build/steps.py:
* Tools/CISupport/ews-build/steps_unittest.py:

Canonical link: https://commits.webkit.org/303565@main
